packaging/linux/openra.appdata.xml

```diff
@@ -1,48 +1,60 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>openra.desktop</id>
-  <pkgname>openra</pkgname>
-  <name>OpenRA</name>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
Technically a software screenshot is licensed under the terms the software is released. https://commons.wikimedia.org/wiki/Template:Free_screenshot As the Westwood / EA owned assets are making up most of the screenshots, those terms apply. If there is nothing for proprietary freeware, then CC0 although incorrect might be the best fit.